_playbook/documents.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: playbook
+title: Creating Accessible Documents
+description: 
+excerpt: 
+sidenav: docs
+categories:
+  - Accessibility
+
+---
+ 
+## Templates
+Templates are used to develop reusable accessible design templates, this helps to reduce the level of effort to create accessible documents each time.
+A template is to be used for font styles and sizes, to have use of already pre-styled structured headings.
+
+### To add a template in Word, Excel or Powerpoint
+1. Click on ‘Tools’, then click ‘Templates and add-ins’ from the dropdown menu. 
```

Comment:
@jaube05 if you change the branch from fork to this repo then we have automation that will search and replace the following quote complaints in the PR. Sorry about these complaints.